src/operation.ts

```diff
@@ -119,7 +118,6 @@ export function getOperationAmountNumber(op: Operation): BigNumber {
     case "OPT_OUT":
     case "REDEEM":
     case "SLASH":
-    case "NFT_OUT":
       return op.value.negated();
```

Comment:
these were mistakes, it makes the UI of live says 0.000000000001 ETH when you send 1 nft. actually it should be "0"